nats.go

```diff
@@ -678,7 +679,7 @@ func (nc *Conn) setupServerPool() error {
 
 	// Place default URL if pool is empty.
 	if len(nc.srvPool) <= 0 {
-		if err := nc.addURLToPool(DefaultURL); err != nil {
+		if err := nc.addURLToPool(DefaultURL, false); err != nil {
```

Comment:
Technically, this one is implicit since we add to the pool if the pool is empty at this point (based on Options.Servers). But agree that in this context we consider "implicit" only the ones returned by the server's INFO protocol messages.